options/locale/locale_en-US.ini

```diff
@@ -785,6 +785,8 @@ audio_not_supported_in_browser = Your browser does not support the HTML5 'audio'
 stored_lfs = Stored with Git LFS
 symbolic_link = Symbolic link
 commit_graph = Commit Graph
+commit_graph.select = Select branches to show...
+commit_graph.hide_pr_refs = Hide PRs
```

Comment:
Can we write out "PRs"? I realizie the menu might geht too large, so maybe it needs to be put into a dropdown.

Reply:
it is too large so I've moved it to the dropdown.